docs/structure.rst

```diff
@@ -1,5 +1,5 @@
-Getting Started
-===============
+Structure of the :mod:`bapsflib` package
```

Comment:
I want this to remain "Getting Started".  When new users come to a documentation page they usually look for a "getting started" page.  This is suppose to be a hub where users can branch to different sections of the documentation.

I don't mind keeping "Structure of the bapsflib package" as a sub-heading.

Reply:
I changed this because it was describing what the package structure was like rather than a getting started page. The page could be recreated and we could make it a hub as you've described. Should I branch out stuff not related to the contribution section and do another pull request for that?